eth/plasma.go

```diff
@@ -20,7 +20,7 @@ type PlasmaClient struct {
 	plasma      *contracts.Plasma
 	privateKey  *ecdsa.PrivateKey
 	userAddress string
-	ethClient   *Client
+	ethClient   Client
```

Comment:
Won't this copy the eth client?

Reply:
Not ethClient is an interface (from a struct).